scheduler/scheduler.go

```diff
@@ -761,9 +761,6 @@ func (s *scheduler) HandleGomitEvent(e gomit.Event) {
 			"event-namespace": e.Namespace(),
 			"task-id":         v.TaskID,
 		}).Debug("event received")
-		// We need to unsubscribe from deps when a task has ended
-		task, _ := s.getTask(v.TaskID)
-		task.UnsubscribePlugins()
```

Comment:
FYI:
This happens before emitting the event - see [scheduler/task.go](https://github.com/IzabellaRaulin/snap/pull/10/files#diff-77f163e381dd113e90ebd1bd76479b54R492)